### PR TITLE
Create harbour-counter-pl.ts

### PR DESCRIPTION
**Pull request by atlochowski**
Polish translation

### CODE REVIEW

File: translations/harbour-counter-pl.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    </message>
+    <message id="counter-switch-sounds">
+        <source>Play sounds</source>
+        <extracomment>Text switch label</extracomment>
+        <translation>Odtwarzaj dźwięki</translation>
+    </message>
+    <message id="counter-button-reset">
+        <source>Reset</source>
+        <extracomment>Button label (resets counter to zeto)</extracomment>
+        <translation>Reset</translation>
```
Comment (monich) [inline translations/harbour-counter-pl.ts]: No translation for Reset?

Comment (monich) [reply translations/harbour-counter-pl.ts]: Actually could be no translation here, "resetować" sounds very much like "reset". Although for Russian I found quite an appropriate a word (сброс) which wasn't a simple transliteration.

Comment (atlochowski) [reply translations/harbour-counter-pl.ts]: In my opinion is better to use word "reset", there is word "reset" in Polish language. I can change that to "Resetuj licznik" - "reset counter".

Comment (atlochowski) [reply translations/harbour-counter-pl.ts]: > Actually could be no translation here, "resetować" sounds very much like "reset". Although for Russian I found quite an appropriate a word (сброс) which wasn't a simple transliteration.

fixed

File: translations/harbour-counter-pl.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    </message>
+    <message id="counter-switch-sounds">
+        <source>Play sounds</source>
+        <extracomment>Text switch label</extracomment>
+        <translation>Odtwarzaj dźwięki</translation>
+    </message>
+    <message id="counter-button-reset">
+        <source>Reset</source>
+        <extracomment>Button label (resets counter to zeto)</extracomment>
+        <translation>Resetuj licznik</translation>
```
Comment (monich) [inline translations/harbour-counter-pl.ts:59]: How about just "Resetuj" to shorten the button text?

File: translations/harbour-counter-pl.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    </message>
+    <message id="counter-label-last_change">
+        <source>Last change:</source>
+        <extracomment>Label text (time and date of the last change)</extracomment>
+        <translation>Ostatnia zmiana:</translation>
+    </message>
+    <message id="counter-label-reset">
+        <source>Reset:</source>
+        <extracomment>Label text (time and date of the last reset)</extracomment>
+        <translation>Reset:</translation>
```
Comment (monich) [inline translations/harbour-counter-pl.ts]: And this one needs a translation too I guess...

File: translations/harbour-counter-pl.ts (hunk 2 above)
Comment (atlochowski) [reply translations/harbour-counter-pl.ts:59]: To be honest, just "Resetuj" is not good. "Reset" or "Resetuj licznik" is ok.

File: translations/harbour-counter-pl.ts (hunk 3 above)
Comment (atlochowski) [reply translations/harbour-counter-pl.ts]: done. this is the final version.

File: translations/harbour-counter-pl.ts (hunk 2 above)
Comment (monich) [reply translations/harbour-counter-pl.ts:59]: OK. I'll have to trust you on this one :)

**Comment by monich** [comment]
Merged, thanks!

**Pull request merged**